src/qibo/hamiltonians.py

```diff
@@ -186,3 +188,129 @@ def MaxCut(nqubits, dense=True):
     if dense:
         return ham.dense
     return ham
+
+
+from qibo.symbols import X as tspX
+from qibo.symbols import Y as tspY
+from qibo.symbols import Z as tspZ
```

Comment:
Another way, which avoids the global import, would be to do
```py
from qibo import symbols
self.X = symbols.X
self.Y = symbols.Y
self.Z = symbols.Z
```
inside `__init__` and then use `self.X`, etc. in the code.